### DISCUSSION TITLE
Always mention specific role when bridged

### DISCUSSION THREAD

**Discussion by EpicLPer** [42wim/matterbridge#1901] [Help / Q&A]
Heya!

Is there any way to always mention a specific role when messages are bridged to Discord?

I'm "abusing" Matterbridge as a way to post announcements from a Telegram Channel to Discord, and want it to always mention a specific role whenever messages are bridged over. I've tried via the NickFormat option and Label, however that didn't seem to work as it only got displayed as standard text in the end.

Thanks!

**Comment by KartikeSingh** [comment]
You can just save the role id in database, and when sending the message add like <@&roleID>
